lib/utils.js

```diff
@@ -362,28 +362,29 @@ module.exports = {
         );
     },
     /*
-     * Returns the first document matching the supplied id in the `patient_id`
-     * field.  Optionally matches a form also.
-     */
+    Gets registration documents for the given ids
```

Comment:
I changed this comment because it was wrong: we neither return documents (instead we return rows) nor do we return the first one (we return all of them)